client/components/Overview/ExpandedPic.jsx

```diff
@@ -1,40 +1,106 @@
 /* eslint-disable react/prop-types */
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import Thumbnail from './Thumbnail.jsx';
 const ExpandedPic = (props) => {
 
     const [order, setIndex] = useState(0);
+    const [zoomed, zoomState] = useState(false);
+    //const [expand, expandState] = useState(true);
 
-    var left = {order} === 0 ? 'hidden' : 'smallLeftArrow';
-    var right = {order} === props.thumbnailArray.length - 1 ? 'hidden' : 'smallRightArrow';
+    var left = order === 0 ? 'hidden' : 'smallLeftArrow';
+    var right = order === props.styleObj.length - 1 ? 'hidden' : 'smallRightArrow';
+    var zoom;
+
+
+    const changeThumbNail = (index)  => {
+      setIndex(index)
+      props.callback(index);
+    }
+
+    const handleRight = () => {
+      console.log(order);
+      var number = JSON.parse(event.target.name);
+      if(number < props.styleObj.length - 1) {
+          props.callback(number + 1);
+          setIndex(order + 1);
+        }
+    }
+
+    const handleLeft = () => {
+      var number = JSON.parse(event.target.name);
+      if(number > 0) {
+          props.callback(number - 1);
+          setIndex(order - 1);
+        }
+
+    }
+
+    const zoomIn = () => {
+      zoomState(!zoomed);
+    }
+
+    const move = (e) => {
```

Comment:
Nice! I like how you made use of event target properties and methods here. Interested to learn more.